src/styles/css/hero.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
+
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-image: url("/src/img/digii.jpg");
```

Comment:
I don't under what the background image is doing on the body. it should be implemented with the img tag on exactly the hero section. please update this